cypress.json

```diff
@@ -0,0 +1,15 @@
+{
+  "baseUrl": "http://127.0.0.1:8080",
+  "fileServerFolder": "tests/e2e",
+  "fixturesFolder": "tests/e2e/fixtures",
+  "integrationFolder": "tests/e2e/integration",
+  "pluginsFile": false,
+  "screenshotsFolder": "tests/e2e/screenshots",
+  "supportFile": "tests/e2e/support",
+  "videosFolder": "tests/e2e/videos",
+  "viewportWidth": 1200,
+  "env": {
```

Comment:
I think we should not put environment variables which are likely to vary for each environment under git control; we should, instead, set environment variables with a file outside git control or --env or CYPRESS_* ([Cypress environment variables](https://docs.cypress.io/guides/guides/environment-variables.html#)). Otherwise, we would have to care that private settings slip into the repository.